# HG changeset patch
# User Rik <rik@octave.org>
# Date 1424396572 28800
#      Thu Feb 19 17:42:52 2015 -0800
# Node ID 4cecd33915efa37f5c42af0a7d60298437660f50
# Parent  a941a65c7cb8dcffc5e0575cdcb71ab16fb5878b
genpropdoc.m: Update axes property name to "mousewheelzoom".

* genpropdoc.m: Update axes property name to "mousewheelzoom".

diff --git a/doc/interpreter/genpropdoc.m b/doc/interpreter/genpropdoc.m
--- a/doc/interpreter/genpropdoc.m
+++ b/doc/interpreter/genpropdoc.m
@@ -523,17 +523,17 @@ the same as the length of 2 units on the
 
       case "fontweight"
       case "gridlinestyle"
       case "interpreter"
       case "layer"
       case "linestyleorder"
       case "linewidth"
       case "minorgridlinestyle"
-      case "mouse_wheel_zoom"
+      case "mousewheelzoom"
         s.doc = "Fraction of axes limits to zoom for each wheel movement."; 
         s.valid = "scalar in the range (0, 1)";
 
       case "nextplot"
       case "outerposition"
         s.doc = "Specify the position of the plot, including titles, \
 axes and legend.  The four elements of the vector are the \
 coordinates of the lower left corner and width and height of the \
